README.Rmd

```diff
@@ -16,10 +16,11 @@ knitr::opts_chunk$set(
 
 # rsketball
 
-##Analysis of NBA players from 2001/02 to 2018/19 seasons
+## Analysis of NBA players from 2001/02 to 2018/19 seasons using ESPN NBA
 
-This package is designated for all NBA enthusiasts! This package works to scrap online tabular data from ESPN NBA website and uses various functions to create graphs and statistical analysis. These include boxplots, rankings, and a summary statistics table.  
-An example of the 2018/19 player stats can be found in the following url:
+This package is designated for all NBA enthusiasts! This package works to scrap online tabular data from ESPN NBA website into a csv file. It also includes various functions to create graphs and statistical analysis for your interest (such as boxplots, player rankings by stats, and a summary statistics table).  
```

Comment:
I think the correct word here is to scrape?

This package works to scrape online tabular data ...

Reply:
ahh yes thanks! will edit it!

Reply:
corrected changes (scrap to scrape in README) made with latest commit "96d44fb".